runtime/shared_common/OSCache.cpp

```diff
@@ -825,6 +851,11 @@ SH_OSCache::getCacheStatistics(J9JavaVM* vm, const char* ctrlDirName, const char
 		return -1;
 	}
 
+	if ((result->layer = getLayerFromName(cacheNameWithVGen)) > J9SH_LAYER_NUM_MAX_VALUE) {
+		Trc_SHR_OSC_getCacheStatistics_LayerNumberTooBig();
```

Comment:
Assert? In theory this can't happen, unless a user creates a cache file name manually.

Reply:
Replaced with an assertion.